File: submissions/sm_021_piyush-jain/week_3/day_3/session2/count_consonants.js
var string="bcdfgh"
var count=0
var arr=string.split("")
for(var i=0;i<arr.length;i++)
{
    if(arr[i]!='a'||'e'||'i'||'o'||'u')
    {
        count=count+1
    }
}
console.log(count)

File: submissions/sm_112_soumik/week_13/day_4/count_consonants.py
check = input("Enter a message seperated my spaces ! :").split()
arr = []
# splitting input and joining without spaces
strg = ''
for letter in check:
    arr.append(letter)

strg = "".join(arr)

vowels = ['a', "e", "i", "o", "u"]
count = 0
# counting the number of consonants
for letter in strg:
    if letter in vowels:
        pass
    else:
        count += 1
print(count)
